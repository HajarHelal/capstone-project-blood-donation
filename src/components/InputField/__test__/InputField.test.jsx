import React from 'react';
import renderer from 'react-test-renderer';
import InputFiled from '../InputField';

// This is an example of a Snapshot. It will create a new snapshot which all future
// tests will use to compare to.
it('renders correctly', () => {
  const tree = renderer
    .create(
      <InputFiled
        placeholder="Name"
        onChange={() => console.log('send')}
        inputValue="name"
        inputName="name"
        Inputtype="text"
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
