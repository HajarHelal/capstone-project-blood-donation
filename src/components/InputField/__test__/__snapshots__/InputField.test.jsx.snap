// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex-1 flex flex-col  mb-4 "
>
  <label
    className="flex-1 mb-2 text-sm sm:text-base font-semibold"
    htmlFor="name"
  />
  <input
    className="flex-1  mr-4 p-2 text-sm sm:text-base tracking-normal  border-2 border-gray-300 focus:outline-none  bg-white placeholder-gray-500"
    id="name"
    name="name"
    onChange={[Function]}
    placeholder="Name"
    type="text"
    value="name"
  />
</div>
`;
